xce/lib/XChemThread.py

```diff
@@ -3235,10 +3228,12 @@ def parse_file_system(self):
             self.Logfile.insert("%s: checking auto-processing results" % xtal)
             self.createSampleDir(xtal)
 
-            if self.target == "=== project directory ===":
-                runDir = os.path.join(collected_xtals, "processed", "*")
-            else:
-                runDir = os.path.join(collected_xtals, "*")
+            runDir = (
```

Comment:
I thought the normal if/else was quicker to read and took less thinking than the if-condition-assignment (or whatever it is called).  
Maybe it's just me.

Reply:
The general argument for conditional assignment is that at a glance you know your variable (`runDir`) is being set, with further reading required to find out what to. This eliminates the cognitive load of examining branches to check it is always set (though that's pretty trivial in this case). Plus, static analysers much prefer it